Tests/ClasslibraryAsSdk/ClasslibraryAsSdk.csproj

```diff
@@ -32,6 +32,10 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'xamarin.ios10'">
+    <NativeReference Include="Resource1.resx" />
```

Comment:
The actual contents of the file is unimportant because it is only used at app link-time. Also, in order for this test to pass/fail, you need to `pack`. The issue does not arise in `build` because iOS never references the "generated" file. But, the `pack` target does when it tries to pack it.